install_config/aggregate_logging.adoc

```diff
@@ -2077,8 +2077,8 @@ To add custom fields to Kibana Visualize:
 . Add custom fields to an Elasticsearch index template:
 
 .. Determine which Elasticsearch index you want to add the fields to, either
-the `.operations.\*` or the `project.*` index. 
-If there is a specific project that has the custom fields, 
+the `.operations.\*` or the `project.*` index.
+If there is a specific project that has the custom fields,
```

Comment:
I see that a lot of the changes to this file are spaces removed at the end of sentences. Wondering if the space after "the custom fields,` `" is needed since the sentence continues on the next line.

Reply:
Looking at the preview output, I think you can disregard my previous comments about the spaces being removed.

Reply:
Yeah these diffs are actually trailing whitespace that was removed - automatically i assume by Atom or some other editor. Trailing whitespace will be ignored in builds, so it's not very important either way, but good practice remove trailing whitespace with a setting in most editors.